src/main/resources/io.vertx.vertx-mongo-embedded-db.json

```diff
@@ -4,7 +4,8 @@
   "options": {
     "worker": true,
     "config": {
-      "port": 27018 // We choose this so as not to clash with default port 27017 of real Mongo
```

Comment:
I think the comment was useful, maybe put back in..?

Reply:
I wasn't sure it would parse correctly anyway (comments in JSON? It should have been a problem all along), but seeing no other errors I put it back in.